packages/button/src/vaadin-button.js

```diff
@@ -50,6 +50,20 @@ class Button extends ActiveMixin(TabindexMixin(FocusMixin(ElementMixin(ThemableM
     return 'vaadin-button';
   }
 
+  static get properties() {
+    return {
+      /**
+       * Indicates whether the element can be focused and where it participates in sequential keyboard navigation.
+       *
+       * @override
+       * @protected
+       */
+      tabindex: {
+        value: 0
```

Comment:
I think this will affect the `vaadin-select` refactoring, which introduces a button mixin: https://github.com/vaadin/web-components/pull/3612/files#diff-936c1ce6ea70b0b86633d2f358c5c3cfa916ecfe6ad5376b5244ddbb04002315R18

If we want to apply `tabindex=0` to the `vaadin-select-value-button` as well (which we do currently), we have to move this to the new button mixin. It looks like the snapshot tests for select should cover this,  but just wanted to mention it regardless.

Reply:
Good point, thanks for the reminder.